frontend/src/components/LedgerOperations/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useMemo } from "react";
+import { useDispatch } from "react-redux";
+
+import "./styles.scss";
+
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus, Network } from "types";
+import { fetchLedgerOperations } from "ducks/ledgers";
+import { SectionCard } from "components/SectionCard";
+import { VerticalBarChart } from "components/charts/VerticalBarChart";
+import {
+  ledgerTransactionHistoryConfig,
+  networkConfig,
+} from "constants/settings";
+
+// max operations per month = 12*1000*60*24*30 = 518400000
+const limitLine = 518400000;
```

Comment:
actually we should lower this number to the max per day, not per month, since that's what this chart is showing
(@jcx120 correct me if I'm wrong)

so instead it would be
```
// max operations per day = 12*60*24*1000 = 17,280,000
const limitLine = 17280000;
```

Reply:
The Max ops line is per month. The X axis is currently shown in the staging instance as daily but should be monthly per the original example in the [doc](https://docs.google.com/document/d/1J2HruGsFGluNSY6IXJ9ooSnIYW5gv_LM-udYnIkkoaw/edit).